browserscripts/prepostscripts/startPerfStats.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Can you fold this "start" script into the module definition for the privileged script, so that we have one module with `start`, `function`, `stop` functions?  Then all the relevant definitions are in one place, and it's probably more clear that the start scripts won't run without the function being run, etc.